test/src/api/routes/test-edition-group.js

```diff
@@ -151,3 +161,83 @@ describe('GET /EditionGroup', () => {
 	 });
 });
 
+
+describe('Browse EditionGroup', () => {
+	// Test browse requests for Edition Group
+	// eslint-disable-next-line one-var
+	let edition;
+	before(async () => {
+		await truncateEntities();
+		edition = await createEdition();
+		const editionGroupBbid = getRandomUUID();
+		const editionGroupAttribs = {
+			bbid: editionGroupBbid,
+			typeId: 1
+		};
+		await createEditionGroup(editionGroupBbid, editionGroupAttribs);
+		// create a revision which adds these two edition in the editionGroup
+		const editor = await createEditor();
+		const revision = await new Revision({authorId: editor.get('id'), id: random.number()})
+			.save(null, {method: 'insert'});
+
+		edition.set('revisionId', revision.get('id'));
+		edition.set('editionGroupBbid', editionGroupBbid);
+		await edition.save(null, {method: 'update'});
+	});
+
+	it('should return list of EditionGroups associated with the edition', async () => {
+		const res = await chai.request(app).get(`/edition-group?edition=${edition.get('bbid')}`);
+		await browseEditionGroupBasicTests(res);
+		expect(res.body.editionGroups.length).to.equal(1);
+	});
+
+	it('should return list of EditionGroups associated with the edition (with Type Filter)', async () => {
+		const res = await chai.request(app).get(`/edition-group?edition=${edition.get('bbid')}&type=Edition+Group+Type+1`);
+		await browseEditionGroupBasicTests(res);
+		expect(res.body.editionGroups.length).to.equal(1);
+		expect(_.toLower(res.body.editionGroups[0].entity.editionGroupType)).to.equal('edition group type 1');
+	});
+
+	it('should return list of EditionGroups associated with the edition (with Type Filter)', async () => {
```

Comment:
Same test description as the test above.

Reply:
I also didn't see any test like that for the other entity types browse endpoints with non-existing types